players_changelog

```diff
@@ -12,6 +12,17 @@ Version 1.13.5+dev:
      * Tweaked the balance of Scenario 2.
    * Delfador's Memoirs:
      * S9: Resolved inability to end level even when Delfador has the Staff (bug #24951)
+   * Heir to the Throne:
+     * Entire campaign: Major cleanup for bugs and logic errors.
+     * S05b ('Isle of the Damned'): Temples randomized, Moremirmu can now appear in any.
+     * S06 ('The Siege of Elensefar'): The thieves are more logical when joining Konrad.
+     * S07 ('Crossroads'): The number of ambushes reduces for enemy leaders killed in S01
+       ('The Elves Besieged').
+     * S13 ('Dwarven Doors'): Added snowfall based upon how turns to complete S12
+       ('Northern Winter').
```

Comment:
Both here and two lines up, I don't like the line break right before the parenthesis. I think the scenario number and its parenthesized name should be treated as an unbreakable unit. In the one case, possibly drop down the word "in" as well.